.gitlab-ci.yml

```diff
@@ -72,6 +72,7 @@ test py38:
     SLURM_CONSTRAINT: gpu
     SLURM_JOB_NUM_NODES: 1
     SLURM_TIMELIMIT: 120
+    NUM_PROCESSES: 3
```

Comment:
What does this cause in the gitlab-ci? Does this PR have any impact on gitlab CI at all? It's all going to end up on the same GPU, though it could be shared I guess.

Reply:
Right. I believe the API calls will time-share the GPU, so in theory it should be fine. It looks like it passes in practice too :-)

Reply:
We could use `auto` here, but I'm afraid of `~24` compilation processes potentially running over the memory limit.

Do you think we should go a bit higher?

Reply:
I think `auto` will pick physical cores, so would be 12. But with tox tests still running 35 minutes, the usefulness of shorter run times becomes less, even if scaling stays perfect. If you have time to play with it a bit today, I'm happy to merge whatever you pick tomorrow.